ui/src/components/Section/section.scss

```diff
@@ -58,6 +58,12 @@
     }
   }
 
+  &.section--banner {
+    z-index: 1;
+    background: var(--funkyGradient);
+    padding: var(--largestSpace);
+  }
```

Comment:
I dont see anything new in this mockup that would require a new class like this. All we are doing is a section with a theme class of funky and just not passing the isSkewed property.